### DISCUSSION TITLE
Script sharing

### DISCUSSION THREAD

**Discussion by aerwen** [256shadesofgrey/eso-light-attack-weave#26] [General]
I have a similar script located here:

https://github.com/aerwen/eso-auto-weaver/blob/master/eso_auto_weave.ahk

Feel free to use any of it.

**Comment by chipeater92** [comment]
hey i was wondering if you had anything for afk farming as when i press loop my dude goes in different directions lol

**Comment by chipeater92** [comment]
ps4 remote macro btw mate forgot to mention
